components/demo/demo-page.js

```diff
@@ -67,9 +80,19 @@ class DemoPage extends LitElement {
 		const classes = {
 			'no-scroll': this._noScroll
 		};
+		const selectedLanguageCode = getDocumentLocaleSettings().language;
+		const languageOptions = supportedBaseLocales.map((code) => {
```

Comment:
This outputs the not super friendly language code... I could also add something to the Intl library that exposes a friendly list of locale names. If we like this approach, I can do that.